.github/workflows/maven.yml

```
@@ -21,12 +21,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Get Java Version
         run: |
           Java_Version=$(mvn help:evaluate "-Dexpression=maven.compiler.release" -q -DforceStdout | sed -e 's/^1\./1.0./')
           echo "Java_Version=$Java_Version" >> $GITHUB_ENV
-      - name: Set up JDK 19
+      - name: Set up JDK 21
```

Maybe we can use the Java_Version env variable to always print the java version we have in pom.xml and which will be installed?
```suggestion
      - name: Set up JDK ${{ env.Java_Version }}
```

---

Sounds good! Did not think about using it there to. 👍